packages/scripts/src/findFaq.ts

```diff
@@ -0,0 +1,139 @@
+import { strict as assert } from "assert";
+import { Db, ObjectId } from "mongodb";
+
+import {
+  Conversation,
+  SomeMessage,
+  AssistantMessage,
+  FunctionMessage,
+} from "mongodb-chatbot-server";
+import { clusterize, DbscanOptions } from "./clusterize";
+import { findCentroid } from "./findCentroid";
+
```

Comment:
nit: i think you could make a new type for the intersection `AssistantMessage | FunctionMessage`:

```ts
type ResponseMessage = AssistantMessage | FunctionMessage;
```